webapp/src/components/channel_header_button/channel_header_buttons.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, { FunctionComponent } from 'react';
+
+type HighlightProps = {
+    shouldHighlight: boolean
+};
+
```

Comment:
I think we can just calls this `Props`

```suggestion
type Props = {
    shouldHighlight: boolean;
};

```